pepper-apis/parent-pom.xml

```diff
@@ -94,7 +94,7 @@
             <!-- configuration -->
             <groupId>com.typesafe</groupId>
             <artifactId>config</artifactId>
-            <version>1.3.1</version>
+            <version>1.4.1</version>
```

Comment:
Was having problems with HOCON. Updated to try to resolve. Unlikely that upgrade helped, but might as well get latest.